City-of-Helsinki/hel-fi-drupal-grants
# Feature/au 290 avustuksen palvelukuvaus

Create Service page node type & build type for showing "palvelukuvaussivut" data from TPR integration.

To test out:
- git pull
- run `make fresh`
- run `make migrate-tpr`
- add terms for taxonomy -> /admin/structure/taxonomy/manage/avustuslaji/overview + /admin/structure/taxonomy/manage/target_group/overview
- add service page -> /fi/node/add/service
- select "Kaupunginhallitus, yleisavustuslomake" for webform, errand service & service channel
- save service page node
- got to /etsi-avustusta -> you should see service page in view and be able to click links to service page & form if you're authenticated with tunnistamo.
